src/poker.js

```diff
@@ -1,10 +1,117 @@
-function winningPair() {}
+// Core criteria
+
+function winningPair(pair1, pair2) {
+  const rankValues = {
+    '2': 2,
+    '3': 3,
+    '4': 4,
+    '5': 5,
+    '6': 6,
+    '7': 7,
+    '8': 8,
+    '9': 9,
+    '10': 10,
+    'J': 11,
+    'Q': 12,
+    'K': 13,
+    'A': 14
+  }
+  const r10 = rankValues[pair1[0]]
+  const r11 = rankValues[pair1[1]]
+  const r20 = rankValues[pair2[0]]
+  const r21 = rankValues[pair2[1]]
+  if (r10 === r11 && r20 === r21) {
+    if (r10 === r20) return []
+    return r10 > r20 ? pair1 : pair2
+  }
+  if (r10 !== r11 && r20 !== r21) return []
+  return r10 === r11 ? pair1 : pair2
```

Comment:
Rather than using vague variable names like `r10`, I'd recommend storing the boolean evaluations in variables instead:

```js
const hand1IsPair = rankValues[pair1[0]] === rankValues[pair1[1]]
const hand2IsPair = rankValues[pair2[0]] === rankValues[pair2[1]]

if (hand1IsPair && hand2IsPair) {
```